docs/PassTemplates/index.rst

```diff
@@ -6,4 +6,4 @@ Pass Templates
     
     Types/index
 
-TODO: Insert pass template text text sdf sdfsdf sdfsdf sd fsdf
\ No newline at end of file
+Sorry! This part of the documentation not ready yet.
```

Comment:
missing "is", not that it matters much though :)
Sorry! This part of the documentation is not ready yet.

Reply:
Same goes for other files with this string